README.md

```diff
@@ -65,6 +65,9 @@ If these conditions are met, feel free to open a PR!
 * **[react-bootstrap/react-bootstrap](https://github.com/react-bootstrap/react-bootstrap)** by [Alexander Shemetovskiy](https://github.com/alexkval)<br>
   ✈️ Bootstrap components built with React.
 
+* **[redoc](https://github.com/Redocly/redoc)** by [Roman Hotsiy](https://github.com/RomanHotsiy)<br>
+  Redoc is an open-source tool for generating documentation from OpenAPI (fka Swagger) definitions.
```

Comment:
```suggestion
  Redoc is an open-source tool for generating documentation from OpenAPI (aka Swagger) definitions.
```

Reply:
I believe the `fka` (former known as) is appropriate here as Swagger (which is currently used only fo the Swagger 2.x standard)  is a former name of a proprietary specification, upon which OpenAPI 3.x was developed, and OpenAPI 3.x standard is not identical to Swagger at this point of time.

Reply:
Ah, that didn't occur to me.